openlibrary/core/lending.py

```diff
@@ -38,17 +38,25 @@
 
 IA_API_URL = "https://archive.org/services/openlibrary.php"
 AVAILABILITY_API = 'https://archive.org/services/loans/beta/loan/index.php'
+IA_USER_API_URL = "http://archive.org/~mek/auth.php"
+IA_XAUTH_API_URL = 'https://www-jnelson2.archive.org/services/xauthn/'
```

Comment:
As per our talk today, I am going to update each of these urls to be in the config (even those on line 39, 40